static/js/components/search/LearnerResult.js

```diff
@@ -12,8 +12,14 @@ export default class LearnerResult extends React.Component {
     result: { _source: SearchResult }
   };
 
+  static gradeAverageDisplay (program) {
+    return _.has(program, 'grade_average') && _.isNumber(program.grade_average) ?
+      `${program.grade_average}%` :
+      '-';
+  }
+
   render () {
-    const { result: { _source: { profile } } } = this.props;
+    const { result: { _source: { profile, program } } } = this.props;
```

Comment:
can you add `program` to the type declaration for `this.props`? flow complains about this line otherwise.